<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssistantDialog</class>
 <widget class="QDialog" name="AssistantDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>im Assistant</string>
  </property>
  <widget class="QGroupBox" name="grpMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Message</string>
   </property>
   <widget class="QTextEdit" name="txtInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpFriend">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Friend</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lbEmail">
       <property name="text">
        <string>EMail:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lbCertId">
       <property name="text">
        <string>CertID:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="leCertId">
       <property name="font">
        <font>
         <family>Fixedsys</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leEmail">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddFriend">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbNick">
       <property name="text">
        <string>Nick:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leNick">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="txtOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>581</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtCert">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fixedsys</family>
      <pointsize>6</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnEncrypt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Encrypt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecrypt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrypt</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSignOnly">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SingOnly</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSkipStrangers">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>530</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SkipStrangers</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtInput</tabstop>
  <tabstop>leCertId</tabstop>
  <tabstop>txtOutput</tabstop>
  <tabstop>btnEncrypt</tabstop>
  <tabstop>cbSignOnly</tabstop>
  <tabstop>cbSkipStrangers</tabstop>
  <tabstop>btnDecrypt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
